capella_rm_bridge/changeset/snapshot_input.schema.json

```
@@ -58,28 +58,12 @@
               "additionalProperties": false,
               "patternProperties": {
                 ".": {
-                  "description": "The values that are available for this EnumerationDataTypeDefinition. They are sometimes called \"options\"",
-                  "type": "object",
-                  "properties": {
-                    "long_name": {
-                      "description": "Value of the \"Long Name\" field in Capella. Here it is the long name of the EnumerationDataTypeDefinition.",
-                      "type": "string"
-                    },
-                    "values": {
-                      "description": "EnumValues in Capella.",
-                      "type": "array",
-                      "items": {
-                        "type": "object",
-                        "properties": {
-                          "id": {
-                            "description": "Value of the \"Identifier\" field in Capella. Here it is the identifier of the EnumValue.",
-                            "type": "string"
-                          },
-                          "long_name": {
-                            "description": "Value of the \"Long Name\" field in Capella. Here it is the long name of the EnumValue.",
-                            "type": "string"
-                          }
-                        }
+                  "description": "The values that are available for this EnumerationAttribute. They are sometimes called \"options\"",
+                  "type": "array",
+                  "items": {
+                    "patternProperties": {
+                      ".": {
+                        "type": "string"
                       }
                     }
                   }
```

If I'm not mistaken, this schema would ask for:

```yaml
data_types:
  something:
    - some_key: a string
      other_key: another string
    - third_key: more string
```

What we actually have in the tests is:

```yaml
data_types:
  something:
    - a string
    - another string
```

Which would mean, in terms of a schema:

```suggestion
                  "items": {
                    "type": "string"
                  }
```